docs/product/ai-in-sentry/seer/index.mdx

```diff
@@ -60,8 +60,12 @@ Seer is a powerful debugging agent, with access to a wide variety of data source
 To start using Seer's capabilities:
 
 1. **Enable Seer**: Make sure AI features are enabled in your [organization settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/#hideAiFeatures)
-2. **Set up GitHub Integration**: For Seer to be able to create patches and open pull requests, [connect your GitHub repositories](/organization/integrations/source-code-mgmt/github/)
-3. **Configure Settings**: Customize Seer's behavior in your project settings
+2. **Set up GitHub Integration**: For issue fix capabilities, [connect your GitHub repositories](/organization/integrations/source-code-mgmt/github/)
+3. **Configure Settings**: Customize Seer's behavior in your [project settings](https://sentry.io/settings/seer/). Seer is enabled by default using these settings:
```

Comment:
This link doesn't work for me. It asks me to sign in, and then it brings me to the *organization* settings. I don't think there's a way to get a deep link to a project's setting, so maybe just document the steps?